hyperspy/_components/eels_cl_edge.py

```diff
@@ -93,6 +93,9 @@ def __init__(self, element_subshell, GOS=None):
                             'fine_structure_coeff',
                             'effective_angle',
                             'onset_energy'])
+
+        self._whitelist['element_subshell'] = ('init', element_subshell)
+        self._whitelist['GOS'] = ('init', GOS)
```

Comment:
Shouldn't `fine_structure_smoothing` and `fine_structure_active` be whitelisted too?